### ISSUE TITLE
AddSessionManager not in IFeatureManager

### ISSUE THREAD

**Issue by ArieJones** [Azure/AppConfiguration#86]
It looks like IFeatureManagementBuilder does not expose AddSessionManager which I thought was supposed to be for when we wanted feature flags to be set per session. Is this by design right now or should there be some other way we should handle this that I am not seeing in the examples. 

Cheers!

**Comment by jimmyca15** [comment]
@ArieJones thanks for the feedback! We can expose a method like this in the `IFeatureManagementBuilder` in the next release of the feature management library. For now it can be done like this

```
services.AddSingleton<ISessionManager, MySessionManager>();

services.AddFeatureManagement();
```

**Comment by jimmyca15** [comment]
Resolved in version [1.0.0-preview-009000001-1251](https://www.nuget.org/packages/Microsoft.FeatureManagement.AspNetCore/1.0.0-preview-009000001-1251)

[Release Notes](1.0.0-preview-009000001-1251)
